api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -127,6 +127,11 @@ message HttpConnectionManager {
     // Whether to annotate spans with additional data. If true, spans will include logs for stream
     // events.
     bool verbose = 6;
+
+    // Maximum length of the request path to extract and include in the HttpUrl tag. Used to
+    // truncate lengthy request paths to meet the needs of a tracing backend.
+    // Default: 256
+    uint32 max_path_tag_length = 7;
```

Comment:
Please use `google.protobuf.UInt32Value` for defaulted integers.

Reply:
done.